[SOC-2408] Bring forward the Social Profile docs to the public site.

wfehr-va opened the pull request:
## [SOC-2408] Bring forward the Social Profile docs to the public site.

This PR takes what we have for the Social Profile endpoint and model on the `products` branch and brings it to master. I've also polished it up and gone through all of the Pre-Release checks: https://vendasta.jira.com/wiki/spaces/API/pages/1651769533/Pre+Release+Checks

[SOC-2408]: https://vendasta.jira.com/browse/SOC-2408?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

richard-rance reviewed (commented):
As discussed, please update the toc.json and changelog.md. You may also write a guide.

wfehr-va commented:
> As discussed, please update the toc.json and changelog.md. You may also write a guide.

Done, I've added both and fixed up a few minor points. Thanks! :+1:

richard-rance commented on docs/Guides/Social/socialProfiles.md:
```diff
@@ -0,0 +1,64 @@
+
+<!--
+type: tab
+title: Request
```
We should also include a sample response.

richard-rance replied:
Users skimming the guide won't see that until they go into operation details. We want them to be able to see what the values should look like without running the command.

richard-rance commented on openapi/social/social.yaml:
```diff
@@ -0,0 +1,176 @@
+                - instagram
+            status:
+              type: string
+              description: A string enum that describes the current status of this Social Profile.
```
Could you explain what the status field is without using the word status? Providing examples of when a social profile would be in the disabled or active status may help.

richard-rance commented on openapi/social/social.yaml:
```diff
@@ -0,0 +1,176 @@
+      required:
+        - type
+      x-lifecycle:
+        Status: Trusted Tester
```
```suggestion
        Status: trustedTester
```
